web-framework-commons/src/main/webapp/js/alfresco.js

```diff
@@ -8358,6 +8358,17 @@ Alfresco.util.Ajax = function()
             }
          }
 
+         if (serverResponse.status === 0 &&
+             document.referrer.length > 0 &&
+             Alfresco.constants.AIMS_AUTH_SERVER_URL !== undefined &&
+             Alfresco.constants.AIMS_AUTH_SERVER_URL.length > 0 &&
+             document.referrer.startsWith(Alfresco.constants.AIMS_AUTH_SERVER_URL)
```

Comment:
Why are you checking the length of the URL and if it is undefined? Why not checking only if it starts with AIMS URL?

Reply:
@cristina-sirbu I'm checking the length if it is different than undefined.

Reply:
@cristina-sirbu Good point, removed the unneeded condition.